src/cli/mod.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright 2020 Contributors to the Parsec project.
+// SPDX-License-Identifier: Apache-2.0
+
+//! Base CLI implementation.
+
+use crate::common::{PROJECT_AUTHOR, PROJECT_DESC, PROJECT_NAME, PROJECT_VERSION};
+use crate::error::ParsecToolError;
+use crate::subcommands::{ParsecToolSubcommand, Subcommand};
+use parsec_client::auth::AuthenticationData;
+use structopt::StructOpt;
+
+/// Struct representing the command-line interface of parsec-tool.
+#[derive(Debug, StructOpt)]
+#[structopt(name=PROJECT_NAME, about=PROJECT_DESC, author=PROJECT_AUTHOR, version=PROJECT_VERSION)]
+pub struct ParsecToolApp {
+    /// How verbose should we be?
+    #[structopt(short = "v", multiple = true, default_value = "0")]
+    pub verbosity: u8,
```

Comment:
This does not seem to be used in the PR, are you going to implement it later?

Reply:
Yes, I was thinking that verbosity would be useful to control the level of output for some of the more complex operations in the future. It's unused at the moment though. 🙂